src/Compute/Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -569,7 +569,7 @@ private async Task<ResourceConfig<VirtualMachineScaleSet>> SimpleParameterSetOrc
                     instanceCount: _cmdlet.InstanceCount,
                     dataDisks: _cmdlet.DataDiskSizeInGb,
                     zones: _cmdlet.Zone,
-                    ultraSSDEnabled: _cmdlet.EnableUltraSSD.IsPresent,
+                    ultraSSDEnabled: (_cmdlet.EnableUltraSSD == true) ? true : (bool?)null,
```

Comment:
Can New-AzVmss be used to update a vmss? If so, might want to allow users to update the value to false.